src/containers/LearningResourcePage/components/LearningResourceContent.jsx

```diff
@@ -60,6 +65,14 @@ import toolbarPlugin from '../../../components/SlateEditor/plugins/SlateToolbar'
 const byLineStyle = css`
   display: flex;
   margin-top: 0;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+const IconContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 64px;
```

Comment:
Burde dette være en unit som endrer seg etter skjermstørrelse?

Reply:
Mulig. Men div-en inneheld to ikon som har fast størrelse. Så tenker det kan være slik.